recipes/xorg/all/conanfile.py

```diff
@@ -72,6 +72,11 @@ def system_requirements(self):
                            "libxxf86vm", "libxv", "xkeyboard-config", "xcb-util", "xcb-util-cursor"], update=True, check=True)
 
     def package_info(self):
+        if conan_version.major >= 2:
```

Comment:
Why the v2 check?

Reply:
Because v1 does not allow setting these on both top and component-level. I had to add the v2 check to pass the CI build.

Reply:
Oh, TIL. Havent used v1 much so things like these slip by me, thanks for the explanation!